### PR TITLE
MNT Add behat tests

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/473

Requires https://github.com/silverstripe/silverstripe-behat-extension/pull/204 to be merged first

### CODE REVIEW

File: tests/behat/src/FeatureContext.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        assertNotNull($ret, "Widget edit form for {$widgetTitle} was not found");
+        return $ret;
+    }
+
+    /**
+     * @param NodeElement $ancestor
+     * @param string $locator
+     * @return NodeElement|null
+     */
+    private function getDecendantHtmlField($element, $locator)
```
Comment (michalkleiner) [inline tests/behat/src/FeatureContext.php:92]:
```suggestion
    private function getDescendantHtmlField($element, $locator)
```

Comment (emteknetnz) [reply tests/behat/src/FeatureContext.php]: Updated

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Build is sill failing

**Comment by emteknetnz** [comment]
Yeah this seems like an actual regression with comments, unrelated to the added behat tests

**Comment by emteknetnz** [comment]
I simply don't understand why commenting is not working in behat and I've spend far too long trying to get this to work, so will just have to stick to manual testing for this one

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**